proto/profile.proto

```diff
@@ -115,6 +120,36 @@ message Sample {
   // label includes additional context for this sample. It can include
   // things like a thread id, allocation size, etc
   repeated Label label = 3;
+  // Breakdown can hold the individual timestamps and values for a sample that
+  // has aggregated values. The type and unit of each breakdown is defined by
+  // the corresponding entry in Profile.sample_type. It's acceptable to have
+  // less, but not more, breakdown entries than Profile.sample_type entries.
```

Comment:
> It's acceptable to have less, but not more

It may be worth clarifying what having less elements here would actually mean. This is somewhat confusing given the fact that `values` above seem to require to have same number of elements as `Profile.sample_type`

Reply:
Also see slack discussion:

<img width="920" alt="image" src="https://user-images.githubusercontent.com/15000/194561681-b0b047a3-6e00-4f59-980d-c9bfe9204a72.png">

TODO: Figure out what to do here.